metadata.json

```diff
@@ -19,6 +19,10 @@
     {
       "name": "puppet/yum",
       "version_requirement": ">= 0.9.6 < 4.0.0"
+    },
+    {
+      "name": "puppetlabs/yumrepo_core",
+      "version_requirement": ">= 1.0.0 < 2.0.0"
```

Comment:
The `yumrepo` resource is no longer defined in the Puppet language spec but is rather available as a (backward compatible) module.